File: bitget-node-sdk-api/src/lib/model/ws/WsBaseReq.ts
export class WsBaseReq {
    private _op!:string;
    private _args!:object[];


    constructor(op: string, args: object[]) {
        this._op = op;
        this._args = args;
    }

    get op(): string {
        return this._op;
    }

    set op(value: string) {
        this._op = value;
    }

    get args(): object[] {
        return this._args;
    }

    set args(value: object[]) {
        this._args = value;
    }
}

File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/ws/WsBaseReq.java
package com.bitget.openapi.dto.request.ws;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WsBaseReq<T> {
    private String op;
    private List<T> args;
}